presto-iceberg/src/main/java/com/facebook/presto/iceberg/IcebergTableProperties.java

```diff
@@ -24,49 +30,96 @@
 import java.util.Collection;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
 
 import static com.facebook.presto.common.type.VarcharType.VARCHAR;
 import static com.facebook.presto.common.type.VarcharType.createUnboundedVarcharType;
+import static com.facebook.presto.iceberg.IcebergWarningCode.USE_OF_DEPRECATED_TABLE_PROPERTY;
 import static com.facebook.presto.spi.session.PropertyMetadata.booleanProperty;
 import static com.facebook.presto.spi.session.PropertyMetadata.integerProperty;
 import static com.facebook.presto.spi.session.PropertyMetadata.longProperty;
 import static com.facebook.presto.spi.session.PropertyMetadata.stringProperty;
 import static com.google.common.collect.ImmutableList.toImmutableList;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+import static java.lang.String.format;
 import static java.util.Locale.ENGLISH;
+import static org.apache.iceberg.TableProperties.COMMIT_NUM_RETRIES;
 import static org.apache.iceberg.TableProperties.UPDATE_MODE;
 
 public class IcebergTableProperties
 {
+    /**
+     * Please use  {@link TableProperties#DEFAULT_FILE_FORMAT}
+     */
+    @Deprecated
     public static final String FILE_FORMAT_PROPERTY = "format";
+    private static final String DEFAULT_FORMAT_VERSION = "2";
```

Comment:
Move to private static section below the public and protected ones

Reply:
updated